File: Exercícios em Python/Exercício 8-Ref.py
saldo=float(input("Insira seu saldo: "))

if saldo >= 1000:
    tarifa = saldo*0.8

if saldo >= 3000:
    tarifa = saldo*0.5
    
if saldo >= 5000:
    tarifa = saldo*0.25

print(f"A tarifa do seu saldo é: {tarifa:.2f}.")

File: Anotações/Exercícios em C/Exercício 8-Ref.c
#include <stdio.h>

int main()
{
    float tarifa, saldo;
    printf("Insira seu saldo: ");
    scanf("%f",&saldo);
    
    if (saldo >= 1000) 
        tarifa = saldo*0.8;
    if (saldo >= 3000) 
        tarifa = saldo*0.5;
    if (saldo >= 5000) 
        tarifa = saldo*0.25;

    printf("A tarifa do seu saldo é: %2f", tarifa, ".");

    return 0;
}
